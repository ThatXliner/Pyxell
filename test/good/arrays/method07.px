
a = [0...10^^7]

a.erase(7, 10^^7 - 8)
a.erase(5)
print a.pop(), a.pop()

a.erase(-3, 2)
print a

a.clear()
print a
