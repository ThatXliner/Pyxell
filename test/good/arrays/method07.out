9999999 6
[0, 1, 4]
[]
